src/components/AlertBanner/style.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+  display: flex;
+  padding: 8px;
```

Comment:
Can we create spacing variables? just so that we don't use random spacing for margin and paddings?